scripts/zones/Buburimu_Peninsula/mobs/Wake_Warder_Wanda.lua

```diff
@@ -0,0 +1,48 @@
+-----------------------------------
+-- Area: Buburimi Peninsula
+--  MOB: Wake Warder Wanda
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_MAGIC_COOL, 50); -- just one spell to spam
+    mob:addMod(MOD_SILENCERES,20);
+    mob:addMod(MOD_BINDRES,20);
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+end;
+
+-----------------------------------
+-- onMobEngaged
+-----------------------------------
+
+function onMobEngaged(mob,target)
+    mob:setMod(MOD_REGAIN, 25);
```

Comment:
Depends on if mob should have regin while idle (usually no). If it only has it during battle, it needs set on engage and removed on disengage